package.json

```diff
@@ -72,6 +75,7 @@
     "source-map-support": "^0.4.0",
     "standard": "^5.4.1",
     "trace": "^2.0.2",
+    "vinyl": "^1.1.1",
```

Comment:
All these changes are not needed, the original v5.x branch built without issues.